.travis.yml

```diff
@@ -7,12 +7,8 @@ notifications:
   email: false
 node_js:
   - stable
```

Comment:
Should we explicitly test on current `lts`, too? There _may_ be issues with `latest`/`7.0.0` concerning `graceful-fs`.